### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by allejo** [gionkunz/chartist-plugin-accessibility#7]
Any chance a new release could be pushed to NPM which contains the issue fixed in #5?

https://www.npmjs.com/package/chartist-plugin-accessibility

**Comment by shal-xu** [comment]
I am also building a React app which cannot load chartist-plugin-accessibility. A new release will be very helpful, or a way to get around the UMD Loading!

**Comment by gionkunz** [comment]
Almost 1 year later, version 0.0.5 has been released today. Sorry for this delay.
